test/z_cluster_SUITE.erl

```diff
@@ -5,6 +5,8 @@
 -include_lib("common_test/include/ct.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
+-define(NUM_NODES, 3).
```

Comment:
this should probably take the chain list, instead of a number, but that can wait until we extend this test to multiple chains.